onnxruntime/test/providers/cpu/model_tests.cc

```diff
@@ -714,6 +804,12 @@ TEST_P(ModelTest, Run) {
             break;
           }
         }
+        for (char* p : output_names) {
```

Comment:
nit: could this clean up of output_names and output_values be done via RAII too?

Reply:
Thanks. I will do it later.